src/intern_blks/ya_intern.c

```diff
@@ -129,6 +131,71 @@ __attribute__ ((gnu_inline)) inline void ya_setup_prefix_suffix(ya_block_t *blk,
 	}
 }
 
+#include <playerctl/playerctl.h>
+#define CHECK_PLAYER_ERROR(PREFIX) \
+	if (e_##PREFIX) { \
+		ya_block_error(blk, "Player error!"); \
+	}
+
+#define PRINT_PAUSED_MSG() \
+	sprintf(startstr,"%s",blk->internal->option[2]);
+
+void ya_int_song(ya_block_t *blk) {
+	char *startstr = blk->buf;
+	size_t prflen = 0;
+	size_t suflen = 0;
+	ya_setup_prefix_suffix(blk, &prflen, &suflen, &startstr);
+
+	GError *e_tmp;
+	GError *e_title;
+	GError *e_artist;
```

Comment:
To be consistent with the other blocks, all `GError` and `size_t` variables should be declared on one line.